service/key/key.go

```diff
@@ -201,7 +201,7 @@ func AlertManagerKey() string {
 func APIUrl(obj interface{}) string {
 	switch v := obj.(type) {
 	case *v1.Service:
-		return fmt.Sprintf("%s:443", v.Spec.ClusterIP)
+		return "kubernetes.default:443"
```

Comment:
This makes us less reliant on the load balancer IP that can (albeit not often) change

Reply:
i doubt it can ever change, but I still think it's worth the change